mathics/builtin/evaluation.py

```diff
@@ -10,6 +10,8 @@
 
 class RecursionLimit(Predefined):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/RecursionLimit.html</url>
```

Comment:
The link should be https://reference.wolfram.com/language/ref/$RecursionLimit.html